extensions/asset/json-schema/schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "stac-extension-asset.json#",
+    "title": "Asset Definition Extension Specification",
+    "description": "STAC Asset Definition Extension to a STAC Collection",
+    "allOf": [
+      {
+        "$ref": "../../../collection-spec/json-schema/collection.json#/definitions/core"
+      },
+      {
+        "$ref": "#/definitions/asset"
+      }
+    ],
+    "definitions": {
+      "asset": {
```

Comment:
The JSON Schema is incorrect. The asset here is the schema name, but a property asset (which should be required) is missing.

Reply:
All the schemas I've done today are likely wrong. I'll make an attempt to fix.

Reply:
They are still not validating. I took a stab and pushed an updated version of the JSON Schema for your convenience.

PS: When developing, I always use https://www.jsonschemavalidator.net/ to check JSON Schemas against the examples. :-)